screencapture/activator.go

```diff
@@ -53,6 +53,37 @@ func EnableQTConfig(device IosDevice) (IosDevice, error) {
 	return device, err
 }
 
+func DisableQTConfig(device IosDevice) (IosDevice, error) {
```

Comment:
Please add a comment describing the function, like:
//DisableQTConfig disables the video streaming config for the device